kie-server-parent/kie-server-controller/kie-server-controller-rest/src/main/java/org/kie/server/controller/rest/RestSpecManagementServiceImpl.java

```diff
@@ -148,7 +163,12 @@ public Response saveServerTemplate(@Context HttpHeaders headers,
 
             specManagementService.saveServerTemplate(serverTemplate);
             logger.debug("Returning response for save server template with id '{}': CREATED", serverTemplateId);
-            return createCorrectVariant("", headers, Response.Status.CREATED);
+            UriBuilder builder = uriInfo.getAbsolutePathBuilder();
+            builder.path(serverTemplate.getId());
```

Comment:
Maybe I'm missing something, but doesn't the `uriInfo.getAbsolutePathBuilder();` already contain full path of the reuquest including resolved URI path template parameters, e.g. `..../servers/2`? If that's the case, I don't think we need to append yet another ID by calling `builder.path(serverTemplate.getId());`.

Reply:
@MarianMacik good catch, fixed.